tests/test_examples.py

```diff
@@ -0,0 +1,9 @@
+import shlex
+
+
+def test_2bin_singlechannel(tmpdir, script_runner):
+    command = 'pyhf inspect {0:s}'.format(
+        'docs/examples/JSON/2-bin_single-channel.json'
```

Comment:
I initially felt a bit unsure if it was a good idea to have a file that is going to be tested as part of the code live under `docs/` instead of `pyhf/`, but doing so keeps it out of the manifest and I then also remembered that we test the Jupyter notebooks live under `docs/` so this shouldn't be any different.